azure-functions/acbs-function/activity-create-deal-guarantee/index.js

```diff
@@ -14,12 +14,12 @@ const { isHttpErrorStatus } = require('../helpers/http');
 const { findMissingMandatory } = require('../helpers/mandatoryFields');
 
 const mandatoryFields = [
-  'dealIdentifier',
-  'guarantorParty',
+  'effectiveDate',
   'limitKey',
   'guaranteeExpiryDate',
-  'effectiveDate',
   'maximumLiability',
+  'guarantorParty',
+  'guaranteeTypeCode',
```

Comment:
I don't know if this matters or not for DTFS @abhi-markan, but the `POST /deals/{dealId}/guarantees` endpoints doesn't have a request field for `guaranteeTypeCode` so you don't need to send it in the request if you don't want to

Reply:
Have updated.